packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -689,15 +717,17 @@ Or read our docs at https://www.prisma.io/docs/concepts/components/prisma-client
       transaction: PrismaPromiseTransaction | undefined,
       clientMethod: string,
       args: RawQueryArgs,
+      middlewareArgsMapper?: MiddlewareArgsMapper<unknown, unknown>,
     ) {
       return this._request({
         action: 'queryRaw',
         args,
         transaction,
         clientMethod,
-        argsMapper: rawQueryArgsMapper(this, clientMethod),
+        argsMapper: rawQueryArgsMapper(this._activeProvider, clientMethod),
         callsite: getCallSite(this._errorFormat),
         dataPath: [],
+        middlewareArgsMapper,
```

Comment:
Instead of having a new mapper, could the middleware logic re-wrap `argsMapper`?